src/dot-components/Product-listing/product.jsx

```diff
@@ -0,0 +1,197 @@
+import "../../public-css/navbar.css";
+import "./product.css";
+import "../../public-css/root.css";
+
+
+const Products = () => {
+  return (
+    <div className="App">
+        <div class="nav-con">
+        <nav class="nav-bar">
+            <div>
+                <h2 class="website-name"><a href="../../index.html" class="link-style link-color-primary"> D<img src="./images/dot.png" class="dot-img"/>T Store</a></h2>
+
+                <p class="sm-txt"><i>Everything ends with a dot.</i></p>
+            </div>
+            
+            <div class="box">
+                <input type="text" placeholder="Search" class="search-box input"/>
+            </div> 
+            <div class="nav-btn">
+                <button class="btn secondary-btn"><a href="../login/login.html" class="link-style link-color">Login</a></button>
+                <div class="btn-badge">
+                    <button class="btn icon-only-btn"><i class="far fa-user icon"></i></button>
+                    <div class="real-badge">6</div>
+                </div>
+                <div class="btn-badge">
+                    <button class="btn icon-only-btn"><a href="../wishlist/wishlist.html"><i class="far fa-heart icon"></i></a></button>
+                    <div class="real-badge">4</div>
+                </div>
+                
+                <div class="btn-badge">
+                    <button class="btn icon-only-btn"><a href="../cart/cart.html"><i class="far fa-shopping-cart icon"></i></a></button>
+                    <div class="real-badge">1</div>
+                </div>
+            </div>
+        </nav>
```

Comment:
should use the navbar component which is already created

Reply:
Yeah okay, forgot to do that.